shared/actions/chat.js

```diff
@@ -1718,9 +1718,10 @@ function * _sendNotifications (action: AppendMessages): SagaGenerator<any, any>
   const appFocused = yield select(_focusedSelector)
   const selectedTab = yield select(_routeSelector)
   const chatTabSelected = (selectedTab === chatTab)
+  const convoIsSelected = action.payload.isSelected
 
   // Only send if you're not looking at it
-  if (!action.isSelected || !appFocused || !chatTabSelected) {
```

Comment:
my guess is that this is checking for existence, and flow is okay with you doing that. Let me think if there's a way to write this that would have been caught.